README.md

```diff
@@ -807,6 +807,12 @@ Whether two URIs can be considered equivalent. Both URIs are normalized automati
 This of course assumes they will be resolved against the same base URI. If this is not the case, determination of
 equivalence or difference of relative references does not mean anything.
 
+## Version Guidance
+
+| Version | Status         | PHP Version      |
+|---------|----------------|------------------|
+| 1.x     | Security fixes | >= 5.4           |
```

Comment:
this may be what is in the composer.json, but it's actually wrong. let's make this `>= 5.4, < 8.2`. We will never support PHP 8.2 or higher on the 1.x branch, but we will support all PHP 8.x versions on the 2.x series, at least while it is current.

Reply:
Done